lte/gateway/python/dhcp_helper_cli/README.md

```diff
@@ -0,0 +1,25 @@
+# DHCP Helper CLI
```

Comment:
Should a short info be added why this cli tool is refactored in a separate folder/release package?

Reply:
In n month someone might ask why this functionality is put in a separated tool - and if no obvious answer is found then it might get re-integrated.